docs/source/configuration/experimental.md

```diff
@@ -0,0 +1,35 @@
+---
+myst:
+  html_meta:
+    "description": "Enable experimental features in Volto's configuration object"
+    "property=og:description": "Enable experimental features in Volto's configuration object"
+    "property=og:title": "Experimental features"
+    "keywords": "Volto, Plone, frontend, experimental"
+---
+
+# Experimental features
```

Comment:
Is there a relation between "experimental features" and a "canary" release? Thinking out loud here... For example, does a canary release include experimental features? I happened to mention that in another PR:

https://github.com/plone/volto/pull/3934/files#diff-c060614717ad6cda93b44a2e34baa9d862e93884d0318798631e34808ff3842dR11-R13

Reply:
Oh, also "feature flags". Does that belong in here?